src/game/default/g_entity.c

```diff
@@ -71,6 +133,9 @@ static g_entity_spawn_t g_entity_spawns[] = { // entity class names -> spawn fun
 
 	{ "worldspawn", G_worldspawn },
 
+	// compatibility-only entities
+	{ "weapon_chaingun", G_weapon_chaingun },
+
```

Comment:
Because the map won't have any bolts - the function above picks the nearby bullets and turns them into bolts.

Also there isn't a G_weapon_lightning since it's an item which has a different spawn routine.